### ISSUE TITLE
Title input field misses a label

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/calendar#3932]

**Comment by JuliaKirschenheuter** [comment]
I have done the solution with "aria-label", because the real issue is accessibility and it will be well done with "area-label" instead of real "label". Please let me know, if we need a real one and how it should behave together with input (where should it stay and so on).
For the implementation i had to change a bit the original ActionInput. I have a done a additional prop, because otherwise this attribute would stay at the root element of the ActionInput.
